workspaces/redhat-argocd/plugins/argocd/package.json

```diff
@@ -45,11 +45,11 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.61",
     "@mui/icons-material": "^6.0.0",
-    "@patternfly/patternfly": "^5.1.0",
+    "@patternfly/patternfly": "^6.0.0",
     "@patternfly/react-charts": "^7.1.1",
-    "@patternfly/react-core": "^5.1.2",
-    "@patternfly/react-icons": "^5.1.1",
-    "@patternfly/react-tokens": "^5.1.2",
+    "@patternfly/react-core": "^6.0.0",
+    "@patternfly/react-icons": "^6.0.0",
+    "@patternfly/react-tokens": "^6.0.0",
```

Comment:
I think `@patternfly/react-tokens` and `@patternfly/react-charts` are unused dependancies, so we can remove these deps.

Reply:
Yeah they look unused, removed.